### ISSUE TITLE
codedeploy don't support overwrite file in codepipeline

### ISSUE THREAD

**Issue by zjjott** [aws/aws-codedeploy-agent#145]
I have read #14 ,but until now ,there is not support in codepipeline to overwrite file

**Comment by rohkat-aws** [comment]
@zjjott not sure if this is supposed to be a codedeploy issue?

**Comment by huribeir** [comment]
@rohkat-aws I think he meant CodeDeploy...this is around the fact that we currently can't specify Content Options using the appspec file.

**Comment by rohkat-aws** [comment]
@huribeir seems like a feature request to me, if you can just explain it a little bit more. We can look into it ,if its feasible.

**Comment by huribeir** [comment]
Yes definitely a feature request, and documented on #14. Information is that this has been already added as a feature request back in 2015.

**Comment by quentinhayot** [comment]
This feature needs some love guys, come on... It's been 3 years !
When using CodePipeline, you can't really use the CLI to allow files overwriting, can you ?

**Comment by pandeyprateek** [comment]
any update on this feature?

**Comment by RobinCowash** [comment]
Need it so much!

**Comment by quentinhayot** [comment]
Yeah, any news on this ?

**Comment by BartusZak** [comment]
So I do need to remove it every time when I finish deployment...

**Comment by josjaf** [comment]
+1 for this feature!

**Comment by gantta** [comment]
+1 for this feature

**Comment by hanischandrew** [comment]
+1 for this feature

**Comment by teamhide** [comment]
Any new feature here?

**Comment by iddriss** [comment]
Any update here?

**Comment by Helen1987** [comment]
Can be fixed after #14 is fixed

**Comment by Helen1987** [comment]
Should be fixed alongside with #14

**Comment by brblck** [comment]
Duplicate of #14
